tools/python/boutiques/schema/example_good.json

```diff
@@ -0,0 +1,236 @@
+{
+  "name": "ndmg",
+  "tool-version": "v0.0.41-1",
+  "description": "dwi connectome estimation pipeline",
+  "command-line": "ndmg_pipeline [DTI] [BVAL] [BVEC] [MPRAGE] [ATLAS] [MASK] [OUTDIR] [DESIKAN] [JHU] [TALAIRACH] [AAL] [HARVARDOXFORD] [CPAC200] [CLEAN]",
+  "container-image": {
```

Comment:
Tested this - the typo correctly raises an error in the schema. I want `example_good.json` to be an example that passes, so won't propagate this change to the online version.